ftw/file/tests/test_fields.py

```diff
@@ -22,8 +22,8 @@ def test_available_sizes(self):
                        'listing', 'tile', 'preview', 'icon'])
         sizes = self.file.getPrimaryField().getAvailableSizes(self.file).keys()
         sizes = set(sizes)
-        self.assertFalse(expected.symmetric_difference(sizes), "The Available"
-                         " Sizes don't meet the expectations")
+        self.assertTrue(expected.issubset(sizes), "The Available"
```

Comment:
Thank you for the patch, @Tschanzt.

Could we use this commit to make the code a little bit more readable and fix the typos (upper case):

```
self.assertTrue(expected.issubset(sizes), 
                "The available sizes don't include default scales.")
```